src/main/java/core/basesyntax/Machine.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+public abstract class Machine {
+    protected String machine;
+
+    public Machine(String machine) {
+        this.machine = machine;
+    }
+
+    public abstract void doWork();
+
+    public abstract void stopWork();
+
+    @Override
+    public String toString() {
+        return machine;
+    }

```

Comment:
The 'toString' method should return a more informative message. It's better to include a descriptive message that specifies what type of machine it is, rather than just returning the machine field. For example, 'Machine type: ' + machineType.